StreamChat.xcodeproj/project.pbxproj

```diff
@@ -5137,6 +5143,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
+				PRINCIPAL_CLASS = StreamChatUITests.TestEnvironmentSetup;
```

Comment:
any chance something like `"$(PRODUCT_NAME).TestEnvironmentSetup"` would work here? 🤔 not sure if it's possible

Reply:
It most certainly is, great thinking. 🙂